INSTALL.md

```diff
@@ -104,6 +110,12 @@ host-files: build-wasm
 test-wasm:
 	<<test-wasm>>
 
+init-git-hook:
+	<<hook-permission>>
+	<<init-git-hook>>
+
+check: entangle
+	git diff-index --quiet HEAD --
```

Comment:
You have a lot of additions which entangle can also benefit from. We should also make sure that we add those to entangle.

Reply:
Good observation, @jhidding released entangled v1.0.0 which has subcommands. 

* `entangled list ` could be used in the `make clean` command. 
* subcommand to check if all files are tangled aka Markdown and other files are in sync.
* subcommand that can be used in a githook that adds tangled files before pushing

Reply:
Also we could switch from pandoc filter docker command to `entangled tange` docker command.

Reply:
Created https://github.com/entangled/entangled/issues/48, https://github.com/entangled/entangled/issues/49 and https://github.com/entangled/entangled/issues/50